src/cppcache/test/ByteArray.cpp

```diff
@@ -20,7 +20,7 @@
 #include "config.h"
 
 #ifdef _MACOSX
-#include <stdlib.h>  // For wcstombs()
+#include <cstdlib>  // For wcstombs()
 #endif
 #include <wchar.h>
```

Comment:
Ah, that makes sense. Yes cleaning up the public includes is certainly more important.